glib/src/any_gobject.rs

```diff
@@ -0,0 +1,108 @@
+use crate as glib;
```

Comment:
it needs the license header as any file

Reply:
what's the point of this?
```suggestion
```

Reply:
That would be a bug in glib-macros then, I guess